### DISCUSSION TITLE
Upcoming 1.3.0 release

### DISCUSSION THREAD

**Discussion by Baklap4** [performous/performous#773] [Announcements]
Hello everyone a couple of weeks ago a list was shared within [discord (please join)](https://discord.gg/NS3m3ad) about an upcoming 1.3.0 release.

The plan is to create an in between Performous 1.3.0 which should resolve a couple of things:
1. Get the current / upcoming open PR's either closed or merged
    - [ ] https://github.com/performous/performous/pull/533 (Maybe?)
    - [ ] https://github.com/performous/performous/pull/538
    - [ ] https://github.com/performous/performous/pull/541
    - [ ] https://github.com/performous/performous/pull/545 (#766 supercedes this)
    - [ ] https://github.com/performous/performous/pull/591
    - [x] https://github.com/performous/performous/pull/648
    - [x] https://github.com/performous/performous/pull/657
    - [x] https://github.com/performous/performous/pull/721
    - [ ] https://github.com/performous/performous/pull/761 (Maybe?)
    - [ ] https://github.com/performous/performous/pull/766
    - [x] https://github.com/performous/performous/pull/769
2. Make webserver generally available for all OS'ses but primarly windows (currently only available for mac and linux)
    - [ ] https://github.com/performous/performous/pull/766
3. Completion of translations in terms of string which currently aren't captured for translating (PR: 682 by @twollgam )
    - [x] https://github.com/performous/performous/pull/682
4. Make a definitive solution to our dependency structure and how to handle them
    - [ ] Actions TBA
5. Decide on support for Meson build tools
    - [x] Deleted as per: https://github.com/performous/performous/pull/734
6. Basic clean code things (trailing/leading spaces, indents etc)
    - [ ] https://github.com/performous/performous/pull/717
7. Fully automated Quality Check / Build / Deploy by using GitHub-Actions and phasing out Semaphore 1 + 2
    - [x] Removal semaphore buildsystem: https://github.com/performous/performous/pull/735
    - [x] Docker build system (linux) in main repo: https://github.com/performous/performous/pull/842
    - [ ] Add GitHub-Action for static analysis ~ Maybe include in #717 ~

Basicly the goal is to easy our lifes as developers, but also add a couple of new features for the enduser (webserver, translations, new sorting options, better pitch detection and memorizing player names)
🛳️ After that the focus will be shifted towards more features, and faster releasing. 🛳️  More about that later!

As of yet i can't determine a set date, but we're slowly but steady are working on making the Issue/PR list clean again.

Thanks for everyone using our new release, if you encounter errors or have any questions feel free to ask them in [user_support on discord (please join)](https://discord.gg/NS3m3ad) or within the Github Issue list 🎉

**Comment by twollgam** [comment]
I think "Player Screen" https://github.com/performous/performous/pull/757 is not for 1.3 IMHO.

**Reply by Baklap4** [reply]
Removed 757 from the list

**Comment by twollgam** [comment]
@Baklap4 @Lord-Kamina @ooshlablu I want to start the discussion again for a 1.3 release. Could we define the things, issues, prs that must be contained and force then the release?

**Reply by Baklap4** [reply]
I've recently started a draft release, details will soon be shared through discord and then we can make the final llist of issues/prs that must be contained :)

**Comment by Baklap4** [comment]
1.3.0 is released, closing this discussion item thanks for thinking with us people :)
